mysql-test/suite/galera/r/galera_defaults.result

```diff
@@ -42,6 +42,7 @@ WSREP_RSU_COMMIT_TIMEOUT	5000
 WSREP_SLAVE_FK_CHECKS	ON
 WSREP_SLAVE_THREADS	1
 WSREP_SLAVE_UK_CHECKS	OFF
+WSREP_SST_ALLOWED_METHODS	mysqldump,rsync,skip,xtrabackup,xtrabackup-v2
```

Comment:
Not an issue, but as an improvement, can we extend this test case to print the count of wsrep variables we support, similar to the global status; so that it becomes handy for future references?

Something like having
```
SELECT COUNT(*) FROM performance_schema.global_variables WHERE VARIABLE_NAME LIKE 'wsrep_%';
```